gpiozero/boards.py

```diff
@@ -2063,13 +2066,13 @@ def __init__(self, socket=None, initial_value=False, pin_factory=None):
             raise EnergenieSocketMissing('socket number must be provided')
         if not (1 <= socket <= 4):
             raise EnergenieBadSocket('socket number must be between 1 and 4')
-        if initial_value is None:
-            raise EnergenieBadInitialValue("initial value can't be None")
```

Comment:
Just for the sake of tidiness, perhaps it's worth removing this exception from the library? :shrug:

Reply:
...and also from `exc.py` ? (sorry for not being more explicit)

Reply:
Should be removed :) (maybe you are seeing a partial diff?)